src/main/java/li/cil/sedna/gdbstub/GDBStub.java

```diff
@@ -1,13 +1,16 @@
 package li.cil.sedna.gdbstub;
```

Comment:
All of the GDB interop seems to very tightly coupled to the riscv arch now, would it make sense to maybe move the `gdbstub` package into the `riscv` package? Since I find it hard to see where a nice split could be made/making it generic would be overkill at this point.

Reply:
I'd like to make it generic at some point, but for now I can do that